content/en/docs/concepts/overview/working-with-objects/namespaces.md

```diff
@@ -112,3 +112,11 @@ kubectl api-resources --namespaced=false
 ```
 
 {{% /capture %}}
+
+{{% capture whatsnext %}}
+* Learn more about [creating a new namespace](/docs/tasks/administer-cluster/namespaces/#creating-a-new-namespace).
+* Learn more about [deleting a namespace](/docs/tasks/administer-cluster/namespaces/#deleting-a-namespace)
+* See [namespaces design](https://github.com/kubernetes/community/blob/{{< param "githubbranch" >}}/contributors/design-proposals/architecture/namespaces.md).
```

Comment:
the "namespaces.md" part is wrong.

Reply:
Thanks @tengqm for your review. I removed that "namespaces.md" part. Can you please have a look again